### ISSUE TITLE
Acer Aspire 7551 wifi not enabled post update

### ISSUE THREAD

**Issue by arnoldthebat** [arnoldthebat/chromiumos#19]
This machine has a broadcom network adapter and a QualComm Atheros wireless card (PCI).

Running and installing from USB works fine, everything works including web-cam, sound and so on. But then it tells me te reboot because of some updates. Then it happens . . . . no more wifi.

So when I startup with a fresh install, wifi works fine. After the update wifi works strange.

When I startup after the update, wifi shows that is is not enabled. When I connect a lan cable, I get network connectivity and I can work. When I unplug the network cable . . . up comes the wifi networks and I can use a wifi connection.

**Comment by jdezwart** [comment]
hi there,

Here content of /var/log/messages

Just after boot : 
https://drive.google.com/file/d/0B6jct4-OLVXjMm9NMGhxLVlCM3M/view?usp=sharing

just after insterting lan cable : 
https://drive.google.com/file/d/0B6jct4-OLVXjUS1QYW80X3lCTjQ/view?usp=sharing

just after disconecting lan cable : 
https://drive.google.com/file/d/0B6jct4-OLVXjOEw5a0pfeGk5UE0/view?usp=sharing

when starting up without lan cable contected, wifi is disabled and no wifi networks can be seen

after inserting lan cable I have internet connectivity via lan

after unpluging lan cable, the wifi is enabled, wifi networks can be seen and connects to a previous connected wifi network.

when you can' t see the contents of the files let me know,


Jacco

**Comment by jdezwart** [comment]
Hi there Arnold.

I was looking around on my laptop with the strange behaviour and found something that might help you.

when I looked at /var/log/messages and grep all the wlan0 lines, I saw whats in this file : 
https://drive.google.com/file/d/0B6jct4-OLVXjaWh6b2QxUXZyckk/view?usp=sharing

In the first lines, all seems fine and working well. But around line 19 to 21 wlan0 gets deauthenticated by local choise (reason = 3).

So, it works, but not after line 22.

Can this be of any help ?

Jacco
